spectec/src/frontend/multiplicity.ml

```diff
@@ -160,16 +164,18 @@ type occur = Il.Ast.iter list Env.t
 let union = Env.union (fun _ ctx1 ctx2 -> assert (ctx1 = ctx2); Some ctx1)
 
 
+let strip_index = function
```

Comment:
This seems to be defined twice in this module.

Reply:
I defined `strip_index` for both El and Il. Would it better to use distinct names?

Reply:
Ah, sorry, missed that. It's fine then.